common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,276 @@
+		monthly_income_mult = 0.05
+		vassal_tax_contribution_mult = 0.1
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_elite_ranks = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @elite_ranks
+	potential = {
+		##Human
+		this.culture_group = culture_group:human_group
+		NOT = {
+			this = culture:wastewander
+			this = culture:pirate_culture
+		}
+	}
+	modifier = {
+		knight_limit = 1
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_epidemic = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @epidemic
+	potential = {
+		##Undead
+		this.culture_group = culture_group:undead_group
+	}
+	custom = epidemic_conversion_bonus
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_architecture = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @zandalar
+	potential = {
+		##Zandalari
+		this = culture:zandalari
+	}
+	modifier = {
+		build_gold_cost = -0.15
+		build_speed = -0.05
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_saronite_arms = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @saronite
+	potential = {
+		##Undead
+		this.culture_group = culture_group:undead_group
+	}
+	modifier = {
+		skirmishers_damage_mult = 0.1
+		skirmishers_toughness_mult = 0.04
+		massive_infantry_damage_mult = 0.1
+		massive_infantry_toughness_mult = 0.035
+		fort_level = 1
+		build_gold_cost = -0.05
+	}
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+
+####################
+##High Medieval##
+####################
+innovation_tinkers = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @gnome
+	potential = {
+	  ##Gnome
+	  this.culture_group = culture_group:gnome_group
+	}
+	modifier = {
+		monthly_piety_from_buildings_mult = 0.02
+		monthly_prestige_from_buildings_mult = 0.02
+		cultural_head_fascination_mult = 0.1
+		build_speed = -0.05
+	}
+
+	flag = global_regional
+	flag = high_medieval_era_regular
+}
+innovation_arkonite = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @arkonite
+	potential = {
+	  ##Draenei
+	  this.culture_group = culture_group:draenei_group
+	}
+	modifier = {
+		caster_damage_mult = 0.1
+		caster_toughness_mult = 0.05
+		knight_effectiveness_mult = 0.03
+		development_growth_factor = 0.32
+	}
+
+	flag = global_regional
+	flag = high_medieval_era_regular
+}
+innovation_iron_star = {
+	group = culture_group_regional
+	culture_era = culture_era_high_medieval
+	icon = @iron_star
+	potential = {
+	  ##Orc
+	  this.culture_group = culture_group:orc_group
+	}
+	modifier = {
+		movement_speed = 0.03
+		light_cavalry_damage_mult = 0.05
+		light cavalry_toughness_mult = 0.03
+		heavy_cavalry_damage_mult = 0.1
+		heavy_cavalry_toughness_mult = 0.05
+		monthly_income_mult = 0.025
```

Comment:
Only two decimal places allowed.
```suggestion
		monthly_income_mult = 0.02
```
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104783537-aedbf780-579f-11eb-83b8-c57b0c936121.png)

</details>